src/ui_fsmenu/about.cc

```diff
@@ -54,23 +53,42 @@ FullscreenMenuAbout::FullscreenMenuAbout()
 		log_err("%s", err.what());
 	}
 
-	close_.sigclicked.connect([this]() { clicked_back(); });
+	graphic_resolution_changed_subscriber_ = Notifications::subscribe<GraphicResolutionChanged>(
+	   [this](const GraphicResolutionChanged&) { layout(); });
+
+	close_.sigclicked.connect([this]() {
+		end_modal<FullscreenMenuBase::MenuTarget>(FullscreenMenuBase::MenuTarget::kBack);
+	});
+
 	layout();
 	tabs_.load_tab_contents();
 }
 
+bool FullscreenMenuAbout::handle_key(bool down, SDL_Keysym code) {
+	if (down) {
+		switch (code.sym) {
+		case SDLK_KP_ENTER:
+		case SDLK_RETURN:
+		case SDLK_ESCAPE:
+			end_modal<FullscreenMenuBase::MenuTarget>(FullscreenMenuBase::MenuTarget::kBack);
+			return true;
+		default:
+			break;
+		}
+	}
+	return UI::Window::handle_key(down, code);
+}
+
+constexpr int16_t kPadding = 4;
 void FullscreenMenuAbout::layout() {
-	// Values for alignment and size
-	butw_ = get_w() / 5;
-	buth_ = get_h() * 9 / 200;
-	hmargin_ = get_w() * 19 / 200;
-	tab_panel_width_ = get_inner_w() - 2 * hmargin_;
-	tab_panel_y_ = get_h() * 14 / 100;
+	if (!is_minimal()) {
+		set_size(calc_desired_window_width(parent_), calc_desired_window_height(parent_));
+
+		close_.set_size(get_inner_w() / 2, get_h() / 16);
+		close_.set_pos(Vector2i(get_inner_w() / 4, get_inner_h() - kPadding - close_.get_h()));
```

Comment:
`close_` could go in a horizontal box with center alignment. The tab panel should also be boxed up.

This is not new though, so I'll leave it up to you if you want to work on this in this branch or not.